modulemd/v2/include/modulemd-2.0/modulemd-dependencies.h

```diff
@@ -100,6 +100,19 @@ modulemd_dependencies_set_empty_buildtime_dependencies_for_module (
   ModulemdDependencies *self, const gchar *module_name);
 
 
+/**
+ * modulemd_dependencies_clear_buildtime_dependencies:
+ * @self: This #ModuleDependencies
```

Comment:
Should be Module**md**Dependencies. There are four other instances of this same typo--all in this file.

Reply:
Actually, running `grep -r '#Module[A-Z]' ...` on the source directory reveals several more similar typos.

Reply:
I'll open an issue and fix that more globally, then. It's a documentation comment, so it's not particularly urgent.

Reply:
I opened #301 to track that.